core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackend.scala

```diff
@@ -328,6 +327,18 @@ private[spark] class MesosSchedulerBackend(
     }
   }
 
+  // check if all constraints are satisfied
+  //  1. Memory requirements
+  //  2. CPU requirements - need at least 1 for executor, 1 for task
+  def isOfferSatisfiesRequirements(cpusOffered: Double, memory : Double,
+                                slaveId: String, sc : SparkContext): Boolean = {
```

Comment:
Please use consistent style (and similar to the other places in the code). Don't put a space between identifier and `:`. It should be `sc: SparkContext`, not `sc : SparkContext`. There are several occurrences of this style in this PR.